mkdocs.yml

```diff
@@ -62,6 +62,8 @@ pages:
   - Consul integration: Administrator Guide/Consul.md
   - Split brain and ways to deal with it: Administrator Guide/Split brain and ways to deal with it.md
   - Arbiter volumes and quorum options: Administrator Guide/arbiter-volumes-and-quorum.md
+  - Thin Arbiter volumes: Administrator Guide/Thin-Arbiter-Volumes.md
+  - Trash for GlusterFS: Administrator Guide/Trash.md
```

Comment:
Actually, I have not given this name and I am not sure if I should change it. I just changed the numbering.

Reply:
@sankarshan, in my opinion, this document is in good shape. However, documentation is an evolving process and can always be improved. This will take time for discussion and modification.  
I would request you to open a github issue so that we can priorities our work along with this issue and work on that.